include/deal.II/numerics/matrix_tools.h

```diff
@@ -53,6 +53,8 @@ namespace hp
 
 
 #ifdef DEAL_II_WITH_PETSC
+#  include <petscvec.h>
```

Comment:
Ah, I see where you use it further down below. But shouldn't it be enough to `#include` the `petscmath.h` file?